src/routes/Chatting.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import chatListData from '../json/ChatList.json';
+
+export default class Chatting extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            chatList : chatListData,
+            searchFriendName : ''
+        }
+    }
+    render(){
+        const handleSearch = (e) => {
+            e.preventDefault();
+            this.setState({searchFriendName : e.target.value});
+        }
+        const showSearchBar = (
+            <SearchBar placeholder="이름 검색" value={this.state.searchFriendName} onChange={handleSearch}/>
+        )
+        const printChatting = (chats) => {
+            chats = chats.filter((chat) => {
+                return chat.name.indexOf(this.state.searchFriendName) > -1;
+            });
+            return chats.map((chat) => {
+                return (
+                    <Link to={{
+                        pathname: `/chatting/${chat.id}`,
+                        state: {
+                            id : chat.id
+                        },
+                        }}
+                    >
+                        <Chat>
+                            <SenderImg src={chat.image} alt={chat.name}></SenderImg>
```

Comment:
Img 태그는 길게 쓸 필요가 없을 거 같아요! 

```suggestion
                            <SenderImg src={chat.image} alt={chat.name} />
```